src/test/unit/middlewares/idValidation.spec.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require('mongoose')
+const { INVALID_ID } = require('~/consts/errors')
+const { createError } = require('~/utils/errorsHelper')
```

Comment:
I think better way will to mock `createError` with some implementation. And check what obtains this function and that it was called.